hack/test-cmd.sh

```diff
@@ -727,6 +728,110 @@ __EOF__
   kube::test::get_object_assert rc "{{range.items}}{{$id_field}}:{{end}}" ''
 
   ######################
+  # Multiple Resources #
+  ######################
+
+  kube::log::status "Testing kubectl(${version}:multiple resources)"
+  # TODO: add test for types like ReplicationControllerList, ServiceList
+
+  ### Create, get, describe, replace, label, annotate, and then delete service nginxsvc and replication controller my-nginx from YAML, separated by ---
```

Comment:
As much as I dislike bash meta programming, why not make this a loop over three different file suffixes for each test and verify with the exact same steps?  for suffix in (list.json concat.json concat.yaml)